### DISCUSSION TITLE
Add meshlab to user group

### DISCUSSION THREAD

**Discussion by marioteodoru** [cnr-isti-vclab/meshlab#1153] [Q&A]
Hello! 
Anybody having issues with Transfer:Vertex Attributes to Texture on ubuntu? 
I'm trying to transfer the Vertex color from a mesh to low res copy of the same mesh in order to transform the Vertex color into a texture. 
The filter just hangs at 0% rasterising faces.

**Comment by alemuntoni** [comment]
Are uou using the last version of MeshLab (2021.10)? Could you please share the meshes and the exact operations that are you performing in order to help us reproduce the issue?

**Reply by marioteodoru** [reply]
Hello @alemuntoni . Yes, i'm using 2021.10
I think it might be my lack of knowledge in using Meshlab. I'll have to do more study and I'll get back if I don't manage it.
